File: 2.0/php/webhooks/findWebhookResources.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->findWebhookResources();
        $request->getOperationConfig()->setQueryParameters(array("fields" => "items(events)"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

File: 2.0/java/webhooks/findWebhookResources.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.webhooks.WebhooksApi;
import com.callfire.api.client.api.webhooks.model.WebhookResource;

import java.util.List;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        WebhooksApi api = client.webhooksApi();
        List<WebhookResource> resources = api.findWebhookResources("items(events)");
    }
}
